ui2/src/pages/custody/Accounts.tsx

```diff
@@ -46,7 +48,7 @@ const AccountsComponent: React.FC<RouteComponentProps & Props> = ({
 
   const requestCloseAccount = async (c: CreateEvent<AssetSettlementRule>) => {
     const service = clientServices.find(s => s.payload.provider === c.payload.account.provider);
-    if (!service) return; // TODO: Display error
+    if (!service) return displayErrorMessage({ message: 'You do not provide custody services.' });
```

Comment:
I would suggest 

`{ header: 'Failed to close account', message: 'Could not find Custody service contract' }`

and a similar pattern with other similar ones

Reply:
Except instead of "close account", replace with whatever the choice being exercised afterwards was supposed to do